workflows/scrnaseq.nf

```diff
@@ -4,34 +4,36 @@ include { KALLISTO_BUSTOOLS                  } from '../subworkflows/local/kalli
 include { SCRNASEQ_ALEVIN                    } from '../subworkflows/local/alevin'
 include { STARSOLO                           } from '../subworkflows/local/starsolo'
 include { CELLRANGER_ALIGN                   } from "../subworkflows/local/align_cellranger"
+include { CELLRANGER_MULTI_ALIGN             } from "../subworkflows/local/align_cellrangermulti"
 include { CELLRANGERARC_ALIGN                } from "../subworkflows/local/align_cellrangerarc"
 include { UNIVERSC_ALIGN                     } from "../subworkflows/local/align_universc"
 include { MTX_CONVERSION                     } from "../subworkflows/local/mtx_conversion"
 include { GTF_GENE_FILTER                    } from '../modules/local/gtf_gene_filter'
 include { EMPTYDROPS_CELL_CALLING            } from '../modules/local/emptydrops'
+include { GUNZIP as GUNZIP_FASTA             } from '../modules/nf-core/gunzip/main'
+include { GUNZIP as GUNZIP_GTF               } from '../modules/nf-core/gunzip/main'
 include { paramsSummaryMultiqc               } from '../subworkflows/nf-core/utils_nfcore_pipeline'
 include { softwareVersionsToYAML             } from '../subworkflows/nf-core/utils_nfcore_pipeline'
 include { methodsDescriptionText             } from '../subworkflows/local/utils_nfcore_scrnaseq_pipeline'
 include { paramsSummaryLog; paramsSummaryMap } from 'plugin/nf-validation'
+include { getGenomeAttribute                 } from '../subworkflows/local/utils_nfcore_scrnaseq_pipeline'
+
 
 
 workflow SCRNASEQ {
 
     take:
     ch_fastq
-    ch_genome_fasta
-    ch_gtf
 
     main:
 
     protocol_config = Utils.getProtocol(workflow, log, params.aligner, params.protocol)
-    if (protocol_config['protocol'] == 'auto' && params.aligner != "cellranger") {
```

Comment:
Based on the `protocols.json` I think `cellrangerarc` must also be part of this check.

Reply:
Maybe a contains should be fine to include both cases ...? However being explicit here is also maybe a good idea, so that its obvious what applies here.

Reply:
think a `params.alinger in [ 'cellranger', 'cellrangerarc' ]` might make it explicit enough

Reply:
added a patch for that: https://github.com/nf-core/scrnaseq/pull/337